contents/docs/integrations/js-integration.md

```diff
@@ -236,6 +236,41 @@ function signup(email) {
 }
 ```
 
+## Config
+
+When calling `posthog.init`, there are various configuration options you can set in addition to `loaded` and `api_host`.
+
+There are [33 different options](https://github.com/PostHog/posthog-js/blob/master/src/posthog-core.js#L53), most of which you do not have to ever worry about. 
+
+Some of the most relevant options are:
+
+<span class="table-borders">
+
+| Attribute | Type | Default | Description |
+| --- | --- | --- | ------------------------- | ------------------------------------------------------- |
+| `api_host` | String | `https://app.posthog.com` | URL of your PostHog instance. |
+| `autocapture` | Boolean | `true` | Determines if PostHog should [autocapture](#autocapture) events. |
+| `loaded` | Function | `function () {}` (no-op) | A function to be called once the PostHog scripts have loaded successfully. |
+| `capture_pageview` | Boolean | `true` | Determines if PostHog should automatically capture pageview events. |
+| <span class="small-text"> `opt_out_capturing_by_default` </span> | Boolean | `false` | Determines if users should be opted out of PostHog tracking by default, requiring additional logic to opt them into capturing. |
```

Comment:
It does stand out a it with small-text, to be honest I'd keep it all the same size at a cost of space, but it's your call!

Reply:
Tradeoffs of using Markdown...

The problem is that the description field gets annoyingly small when that's normal size. Could do some more extensive trickery but not sure that's worth the time.

<img width="801" alt="Screenshot 2020-09-15 at 13 04 26" src="https://user-images.githubusercontent.com/38760734/93214017-251a1000-f754-11ea-8247-828f90074b7c.png">